ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -68,14 +49,15 @@ Item {
         StatusGridView {
             id: gridView
             anchors.fill: parent
-            model: root.collectiblesModel
+            model: root.areCollectionsLoaded ? root.collectiblesModel : Constants.dummyModelItems
             cellHeight: 229
             cellWidth: 176
             delegate: Item {
```

Comment:
I'm wondering if it is needed this `Item` as parent instead of directly call the `CollectibleView` component?